src/styles/blocks/card.scss

```diff
@@ -0,0 +1,96 @@
+.container {
+  &:not(:nth-child(4n)) {
+    margin-right: 50px;
+  }
+  
+  &:nth-last-child(-n + 4) {
+    margin-top: 48px;
+  }
+
+  &__card {
+    display: flex;
+    flex-direction: column;
+  }
+}
+
+.card {
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  &__img {
+    position: relative;
+    left: 3px;
+    height: 134px;
+    width: 160px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+    color: var(--black);
+  }
+  
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: var(--grey);
+    margin-bottom: 16px;
+  }
+
+  &__stars {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-name {
```

Comment:
Don't divide an element name. Use full name:
```
&__price {

&__price-name {

&__price-cost {
```

Reply:
Tests don't pass

Reply:
Ok, so let it as it